samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -562,7 +584,7 @@ mfxStatus CEncodingPipeline::InitMfxEncParams(sInputParams *pInParams)
 
         m_encCtrl.Payload = m_UserDataUnregSEI.data();
         m_encCtrl.NumPayload = (mfxU16)m_UserDataUnregSEI.size();
-        bCodingOption = true;
+        auto codingOption = m_mfxEncParams.AddExtBuffer<mfxExtCodingOption>();
```

Comment:
Please check if it's needed. IMO, no. But I can be wrong. Please check.

Reply:
Didn`t see impact of this. Removed.